index.js

```diff
@@ -1,15 +1,14 @@
-const url = require('url')
 const isIp = require('is-ip')
 const Multiaddr = require('multiaddr')
 
 /**
  * Convert a URI to a multiaddr
  *
- *        http://foobar.com => /dns4/foobar.com/https
- *  https://foobar.com:8080 => /dns4/foobar.com/tcp/8080/https
- *          ws://foobar.com => /dns4/foobar.com/ws
- *   https://127.0.0.1:8080 => /ip4/127.0.0.1/tcp/8080/https
- *        http://[::1]:8080 => /ip6/::1/tcp/8080/http
+ *        http://foobar.com => /dns4/foobar.com/tcp/80/http
+ *       https://foobar.com => /dns4/foobar.com/tcp/443/tls/http
+ *  https://foobar.com:5001 => /dns4/foobar.com/tcp/5001/tls/http
+ *   https://127.0.0.1:8080 => /ip4/127.0.0.1/tcp/8080/tls/http
+ *        http://[::0]:8080 => /ip6/::0/tcp/8080/http
```

Comment:
Why tho :thinking: For `ip4` (1 line above) we default to localhost, shouldn't we do the same here? (`::1`) 

I know this is a nit, but there is a huge difference if a beginner just copies&pastes `0.0.0.0:5001` vs `127.0.0.1:5001` without thinking, so I always try to avoid "listen on all interfaces" in examples :)

Reply:
👍